docs/includes/guides/migration_3.x.adoc

```diff
@@ -0,0 +1,53 @@
+///////////////////////////////////////////////////////////////////////////////
```

Comment:
I am not sure that we need a general migration document. I would prefer to have separate migration documents for SE and MP.

Reply:
separated